README.md

```diff
@@ -15,12 +15,42 @@ information panel, performance data, and more.
 experience, as well as to serve as a platform for research and UI integration
 with other projects.
 
-### Installation
+## Table of Contents
+
+- [Coq LSP](#coq-lsp)
+  - [Table of Contents](#table-of-contents)
+  - [Installation](#installation)
```

Comment:
Arent' the TOC itself and the topleve Coq LSP redundant?

Reply:
My editor automatically generates the TOC, so rather than fighting it I've let it stay.